File: go/models/Like.go
package models

import "github.com/go-pg/pg"

type TblLike struct {
	tableName struct{} `sql:"tbl_like,alias:t" pg:",discard_unknown_columns"`

	ID         string      `sql:"id,pk"`
	CreateTime pg.NullTime `sql:"create_time"`
	DeleteTime pg.NullTime `sql:"delete_time"`
	Liked      *bool       `sql:"liked"`
	OperatorID string      `sql:"operator_id,notnull"`
	TargetID   string      `sql:"target_id,notnull"`
	TargetType string      `sql:"target_type,notnull"`
}

File: src/main/java/com/shumsg/model/entity/Like.java
package com.shumsg.model.entity;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class Like {
    private String id;

    private String targetType;

    private String targetId;

    private String operatorId;

    private Boolean liked;

    private LocalDateTime createTime;

    private LocalDateTime deleteTime;

}